Description: mitigate a race condition in test_HTTPSConnection_SNI_support
 Fixes FTBFS on ppc64el and other archs, like with 0.27.0-1 in experimental.
Author: Miloslav Trmaƒç
Origin: https://gitlab.com/m2crypto/m2crypto/merge_requests/141/diffs
Applied-Upstream: https://gitlab.com/m2crypto/m2crypto/commit/35f5e5c3

--- a/tests/test_ssl.py
+++ b/tests/test_ssl.py
@@ -251,6 +251,7 @@
             c.request('GET', '/')
             c.close()
         finally:
+            time.sleep(sleepTime)
             out, err = self.stop_server(pid)
         self.assertIn('Hostname in TLS extension: "%s"' % srv_host, out)
 
